### DISCUSSION TITLE
Dap configuration: `Async job failed: None`

### DISCUSSION THREAD

**Discussion by getreu** [helix-editor/helix#10271] [General]
The [installation instructions](https://github.com/helix-editor/helix/wiki/Debugger-Configurations#install-debuggers) worked well for me. The only issue: Codelldb expects the `llvm` libraries in another directory then Debian. This was quickly fixed with some symbolic links:

```bash
# pwd
/usr/lldb
# ls -l
lrwxrwxrwx 1 root root 20 Dec 10 06:14 lib -> /usr/lib/llvm-13/lib
# cd lib
# ls -l
lrwxrwxrwx 1 root root       12 Dec 10 06:32 liblldb.so -> liblldb.so.1
lrwxrwxrwx 1 root root       38 Jun 16 12:00 liblldb.so.1 -> ../../x86_64-linux-gnu/liblldb-13.so.1
```

Second issue was: `No external terminal defined`
This disappeared after setting in `config.toml`:

```toml
[editor]
terminal.command = "bash"
terminal.args = ["-c"]
```
Now I get: `Async job failed: None`

My versions:
```sh
$ hx --version
helix 22.12 (96ff64a8)
$ ls x86_64-linux-gnu/liblldb-13.so.1
$ codelldb -V
1.8.1
```

**Comment by BCMM** [comment]
Bash is a shell as opposed to a terminal.

Something like this works, that is, it causes the program that's being debugged to be displayed in a new terminal window:

```TOML
[editor]
terminal.command = "xterm"
terminal.args = ["-e"]
```

(Insert your favourite terminal emulator in place of `xterm`.)

Trying to use `bash` as the debug terminal will, at best, result in the program's output being spewed over the top of Helix, with no way to give input to the program.

I don't think this was the cause of `Async job failed: None`, but it may have helped to mask that cause.
